lib/board.rb

```diff
@@ -0,0 +1,54 @@
+#data and logic of the Tic Tac Toe game board
+#board state held in @cells(Array)
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "] #holds the current state of the game @cells
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.